clients/client-python/gravitino/dto/rel/expressions/unparsed_expression_dto.py

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from gravitino.api.expressions.unparsed_expression import UnparsedExpression
+from gravitino.dto.rel.expressions.function_arg import FunctionArg
+
+
+class UnparsedExpressionDTO(UnparsedExpression, FunctionArg):
+    """Data transfer object representing an unparsed expression."""
+
+    def __init__(self, unparsed_expression: str):
```

Comment:
In `UnparsedExpressionDTO.java` we have Builder subclass. So I think we can have the builder here.